desktop/ui/src/main/java/org/datacleaner/user/DemoConfiguration.java

```diff
@@ -29,6 +29,7 @@
 public class DemoConfiguration {
 
     public static final String DATASTORE_FILE_CONTACTDATA = "datastores/contactdata.txt";
```

Comment:
Let's remove the old "contact data" datastore now that we have a much better one!